kolibri_exercise_perseus_plugin/downloadTranslations.js

```diff
@@ -0,0 +1,59 @@
+const {
+  localeFile,
+  crowdinLangCode,
+  localLocaleFile,
+  remoteLocaleFile,
+  kolibriProjectId,
+  downloadTranslations,
+  languages,
+} = require('./translations');
+const {
+  kolibriApiKey,
+} = require('../crowdinSecrets');
+const fs = require('fs');
+
+const downloadTranslation = (language) => {
+  // Use the language object above to get relevant codes for the language
+
+  const lang = languages[language].lang;
+
+  const locale = languages[language].locale;
+
+  const crowdinLocale = languages[language].crowdinLocale;
+
+  return new Promise((resolve, reject) => {
+    // Download the po file first
+    const pofileDownloadPromise = downloadTranslations(kolibriProjectId, lang, crowdinLocale, kolibriApiKey);
+
+    pofileDownloadPromise.then(zip => {
+
+      console.log(`Finding translations from Kolibri translation files for ${language}`);
+      let translationFile;
+      zip.getEntries().forEach(file => {
```

Comment:
Is there no way to avoid this loop? It seems very kludgy for a modern library.

Reply:
Oh, I can do a find instead.